static/_app/tools/subwaysurfers/poki-sdk.js

```
@@ -0,0 +1,127 @@
+        if (4 & n && "object" == typeof e && e && e.__esModule) return e;
+        var o = Object.create(null);
+        if (t.r(o), Object.defineProperty(o, "default", {
+                enumerable: !0,
+                value: e
+            }), 2 & n && "string" != typeof e)
+            for (var r in e) t.d(o, r, function(n) {
+                return e[n]
+            }.bind(null, r));
+        return o
+    }, t.n = function(e) {
+        var n = e && e.__esModule ? function() {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)